### DISCUSSION TITLE
my tray is gone and song preview doesn't work

### DISCUSSION THREAD

**Discussion by Soapmakingboi** [pear-devs/pear-desktop#115] [Q&A] [answered]
![Screenshot_302](https://user-images.githubusercontent.com/75842987/103578994-9559c680-4ee8-11eb-951c-ce0bcc079400.png)
My options tray is gone and i have no idea how to fix it.
if anyone knows how to fix it please reply.

**Comment by yodyamahesa** [comment, marked as answer]
I had the same problem too
but i solved it

try to uninstall the app and remove Youtube Music folder in
 C:\Users\User\AppData\Roaming\YouTube Music
 and then reinstall the app
 
 its worked

**Reply by Soapmakingboi** [reply]
thanks man i will try it now

Edit: it worked thank you very much

**Reply by yodyamahesa** [reply]
Nice. My pleasure
